requirements/specifications/metrics/media-metrics.md

```diff
@@ -0,0 +1,293 @@
+
+Once called, the `mediaPause` API **SHOULD NOT** be called again unless a `mediaPlay` call has been made since.
+
+### 4.5. Waiting
+The `Metrics` module **MUST** include a `mediaWaiting` API for denoting when the media stops playing, i.e. frames are no longer being rasterized by the media pipeline, due to an unexpected interuption (see below), e.g. buffering.
+
+The `mediaWaiting` API **MUST** have a required `entityId` parameter to specify what entity from the app catalog has been interrupted.
+
+The remainder of this section outlines guidelines that apps should consider when calling `mediaWaiting`.
+
+The `mediaWaiting` API **SHOULD** be called when an "unexpected interruption" occurs. An "unexpected interruption" is due to underlying physical resources such as network, CPU, or even software bugs, e.g. buffering or other non-user, non-business logic reasons that frames might stop rasterizing.
+
+The `mediaWaiting` API **SHOULD NOT** be called when an "intent to pause" occurs. An "intent to pause" is either an explicit user intent, e.g. clicking a pause control, an indirect user action, e.g. opening a settings UX, or app business logic, e.g. an advertisement is displayed over the content.
+
+The `mediaWaiting` API **SHOULD NOT** be called due to the initial "[intent to play](#42-play)."
+
+Once called, the `mediaWaiting` API **SHOULD NOT** be called again unless a `mediaPlay` call has been made since.
+
+### 4.6. Progress
+The `Metrics` module **MUST** include a `mediaProgress` API for denoting playback progress of the media.
+
+The `mediaProgress` API **MUST** have a required `entityId` parameter to specify what entity from the app catalog the progress is from.
+
+The `mediaProgress` API **MUST** have a required `progress` parameter denoting how much progress has been made.
+
+The remainder of this section outlines guidelines that apps should consider when calling `mediaProgress`.
+
+The `mediaProgress` API **SHOULD** be called when playback progress occurs at a rate of roughly once per 60 seconds.
+
+The `mediaProgress` API **SHOULD NOT** be called after playback stops, e.g. while paused or waiting.
+
+The `mediaProgress` API **SHOULD** be called with an up-to-date `progress` value right before any `mediaPause` call.
+
+The `progress` value **SHOULD** be a number greater than 0 and less than 1 denoting the percent of the content that progress has reached, if the duration of the content is static and known, e.g. VOD content.
+
+The `progress` value **SHOULD** be an integer number of secionds from 1 to 86400 denoting the number of seconds from the "beginning" of the content that progress has reached, if the duration of the content is dynamic or not known, e.g. live content.
+
+The "beginning" of the content **SHOULD** be considered as the start of the broadcast/event/etc., if known, otherwise it **MAY** be the start of the available media, e.g available on a CDN.
```

Comment:
Need some clarification on what `available on a CDN` means.  Would a media pipeline even know this?